Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyDoubleSpinCtrl.cpp

```diff
@@ -85,9 +85,9 @@ namespace AzToolsFramework
         m_pSpinBox->setValue(value);
         m_pSpinBox->blockSignals(false);
 
-        //the value didn't change, so don't send a notify
-        if (!AZ::ClampIfCloseMag(value, oldValue) && notifyLater)
+        if (notifyLater)
```

Comment:
Why isn't the AZ::ClampIfCloseMag necessary here anymore?

Reply:
ClampIfCloseMag returns either value or oldValue so I didn't see why it would be part of the conditional...
`//! ClampIfCloseMag(x, y, epsilon) returns y when x and y are within epsilon of each other (taking magnitude into account).  Otherwise returns x.`

Reply:
I ended up just leaving it how it originally was per @forkercat's suggestion. It looks wrong, but safer to leave it as is.